Classes/Domain/Service/StyleAndJavascriptInclusionService.php

```diff
@@ -95,7 +95,7 @@ protected function build(array $resourceArrayToSort, \Closure $builderForLine)
                 $hash = substr(md5_file($resourceExpression), 0, 8);
                 $resourceExpression = $this->resourceManager->getPublicPackageResourceUriByPath($resourceExpression);
             }
-            $finalUri = $hash ? $resourceExpression . '?' . $hash : $resourceExpression;
+            $finalUri = $hash ? $resourceExpression . (str_contains($resourceExpression, '?') ? '&' : '?') . $hash : $resourceExpression;
```

Comment:
is this really _the_ flow way of doing that? With Flow 9 https://github.com/neos/flow-development-collection/pull/3316